src/main/java/com/chaosthedude/naturescompass/config/ConfigHandler.java

```diff
@@ -37,6 +39,12 @@
 
 			desc = "biomeSize * sampleSpaceModifier = sampleSpace. Lowering this value will increase search accuracy but will make the process more resource intensive.";
 			sampleSpaceModifier = builder.comment(desc).defineInRange("sampleSpaceModifier", 16, 0, 1000000);
+			
+			desc = "The maximum value of a step taken during the biome search.";
+			sampleStepMaximum = builder.comment(desc).defineInRange("sampleStepMaximum", 256, 0, 1000000);
+
+			desc = "Changing this to a non-zero value spaces out samples across the same biome. Doing so may speed up searches at the cost of missing small biomes.";
+			sampleMomentumModifier = builder.comment(desc).defineInRange("sampleMomentumModifier", 0, 0, 1.0);
```

Comment:
I'm unsure about using a 0.0-1.0 scale for the momentum modifier, as it seems a bit difficult for users to understand how this changes things without looking at the code. To me, it would make more sense to start the scale at 1.0, as then you can visualize it as a multiplier.

Reply:
Reworded this to make it clearer that it's a ratio of the last step. For regular gradient descent, I've always set this in the range of [0, 1) (see [this](https://ai.stackexchange.com/questions/6377/why-mlp-momentum-term-must-be-in-the-range-0-1)), though the algorithm here isn't exactly that.